docs/content/en/docs/user-guide/managing-piped/configuration-reference.md

```diff
@@ -255,7 +255,7 @@ Must be one of the following structs:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| hookURL | string | The hookURL of a slack channel. | Yes |
+| hookURL | string | The hookURL of a slack channel. | No |
```

Comment:
We don't need to update this, `/docs` is for the current released version of pipecd (which need the hookURL to perform send message), when we release the new version which contains your patch, the `/docs` will be synced with `/docs-dev` and show the latest docs which fit the piped functional 😄